smt.py

```diff
@@ -351,7 +358,14 @@ def make_smtlib(p: assume_prove.AssumeProveProg) -> SMTLIB:
         source.ExprVar(source.type_bool, p.entry))))
 
     cmds.append(CmdCheckSat())
-    return merge_smtlib(emit_cmd(cmd) for cmd in cmds)
+
+    # HACK: include sel4cp prelude
+    raw_prelude = SMTLIB('(set-logic QF_ABV)')
```

Comment:
Doesn't the prelude file also include this ?

Reply:
Yeah it use to, so I removed the `CmdSetLogic()` above:

https://github.com/math2001/ubc/pull/11/files#diff-0a794a32547247b2c4816921412e3ea3739189464ec52720b6658445eb9b9433L320